_posts/2017-10-05-apprenticeship-at-8thlight.md

```diff
@@ -0,0 +1,213 @@
+
+[jakes 1st post]: http://jakescruggs.blogspot.de/2004/05/my-apprenticeship-may-2004.html
+
+## The application, part 1, online
+
+The first contact between an employer and a prospective employee is mostly the employer reads a CV, meets someone at an event, gets a referral or other opportunity that the two get to know about each other. How the process of pre-selecting applicants for setting up a first online session works at 8th Light, I don’t know. What I understood very well is that those people who find 8th Light deliberately and think this can be the right company for me are those that know about craftsmanship, an apprenticeship program and surrounding topics.
+
+One word on wording: lately “craftsmanship” has received quite some critiques for it’s non-inclusive name. Therefore I will try to be careful in wording, but am open to add improvements. There is no intention of offending anyone here.
+
+The other groups of applicants are those that don’t even start the process. Why? As [Jake had already done][jakes-tictactoe] in 2004, the first task you get assigned is writing Tic Tac Toe. This task is assigned to you before you meet face to face. You work on it and get back to 8th Light with a result that you think is good enough. That is the step where some people loose interest. My guess would be they either don’t know what the apprenticeship will look like or they are looking for jobs (or application processes) that are not so “demanding”.
+
+Reading about the first steps in the application process in [here][crafter-application] sums it all up, and it sumarizes quite well who is the audience:
+> We enjoy working with people who are empathetic, who are passionate about teaching and learning, and who value the work they do. If this approach to software development resonates with you, we'd love to talk.
+
+[crafter-application]: https://8thlight.com/job-opportunities/london-crafter/ 
+[jakes-tictactoe]: http://jakescruggs.blogspot.de/2004/05/my-apprenticeship-may-2004.html
+
+The actual online process is a two-step process. First the applicant gets to know someone from 8th Light and gets a task to solve. Normally its Tic Tac Toe. The programming language is not fix. Later the applicant sends in the code and waits for feedback. There will be a code review process going on internally. I assume, if the code is not devastating a second call will be set up where the code is being discussed. The important thing here is to find out how well an applicant takes feedback. 8th Light wants to find out the ability to learn and to incorporate feedback into the applicant’s work. This will trigger a second round of programming for the applicant, where they apply the learnings and sends back the code again.
```

Comment:
We always provide feedback, we figure if you're going to spend some number of hours solving a problem, we need to reciprocate and provide feedback, even if the process stops at that point (it usually doesn't).